tests/domains/courses/router_test.py

```
@@ -0,0 +1,35 @@
+    assert response.status_code == 200
+    assert ((response.json()[0])['name']) == 'FastAPI 강의'
+    assert True
+
+
+def test_get_register():
+
+    assert False
+
+
+def test_add_course():
+    assert False
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)